docs/android/tips/push_notification.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Android push notifications"
+sidebar_label: "Push notifications"
+---
+
+## Push notifications setup
+
+If you want to work on push notifications or use a development build with push notifications, follow these steps:
```

Comment:
This setup is somewhat complex and not always required to work on push notifications because you are probably able to use the WebSocket connection to get notifications for development. The introduction suggestion you need FCM may scare contributors that want to add/change something simple, so the WS option should be mentioned more prominently in my opinion.

Reply:
> because you are probably able to use the WebSocket connection to get notifications for development

Agreed because adding new parameters and commands can all be done on websockets now however the user needs to turn it on to test new fields otherwise they wont work.

Reply:
> the user needs to turn it on to test new fields otherwise they wont work

No, the developer needs to add it to the allowlist in the function. They could deploy it to their project to test, but that isn't described here (could be useful, but also very specific/advanced).

Reply:
I'm going to be more explicit here